plugins/out_stackdriver/stackdriver.c

```diff
@@ -437,7 +504,19 @@ static int stackdriver_format(const void *data, size_t bytes,
          *  "timestamp": "..."
          * }
          */
-        msgpack_pack_map(&mp_pck, 3);
+        if (ctx->severity_key
```

Comment:
Stackdriver documentation says the severity is ENUM type:

https://cloud.google.com/logging/docs/reference/v2/rest/v2/LogEntry#LogSeverity

here you are assigning strings so the solution is not valid.

Reply:
I refactored the code, it is now using int enums. I also have peaked into official fluentd stackdriver plugin from google. I used enum mappings 1:1 from there, with possible severity levels.